Gallery/Vectors/NCL_vector_1.py

```diff
@@ -72,80 +95,68 @@
                scale=75,
                zorder=2)
 
-# Turn on continent shading
-ax.add_feature(cartopy.feature.LAND,
-               edgecolor='lightgray',
-               facecolor='lightgray',
-               zorder=1)
+# Use geocat-viz utility function to format title
+gv.set_titles_and_labels(ax,
+                         maintitle='',
+                         maintitlefontsize=18,
+                         lefttitle="Sea Surface Temperature",
+                         lefttitlefontsize=18,
+                         righttitle="C",
+                         righttitlefontsize=18,
+                         xlabel=None,
+                         ylabel=None,
+                         labelfontsize=16)
+
+# Format tick labels as latitude and longitudes
+gv.add_lat_lon_ticklabels(ax=ax)
+
+# Use geocat-viz utility function to customize tick marks
+gv.set_axes_limits_and_ticks(ax,
+                             xlim=(65, 95),
+                             ylim=(5, 25),
+                             xticks=range(70, 95, 10),
+                             yticks=range(5, 27, 5))
 
-# Define levels for contour map (24, 24.1, ..., 28.8, 28.9)
-levels = np.linspace(24, 28.9, 50)
+# Remove degree symbol from tick labels
+ax.yaxis.set_major_formatter(LatitudeFormatter(degree_symbol=''))
+ax.xaxis.set_major_formatter(LongitudeFormatter(degree_symbol=''))
 
-# Import an NCL colormap, truncating it by using geocat.viz.util convenience function
-gv.truncate_colormap(cmaps.BlAqGrYeOrReVi200,
-                     minval=0.08,
-                     maxval=0.96,
-                     n=len(levels),
-                     name='BlAqGrYeOrReVi200')
-
-# Contourf-plot the SST data
-cf = sst.plot.contourf('lon',
-                       'lat',
-                       extend='both',
-                       levels=levels,
-                       cmap='BlAqGrYeOrReVi200',
-                       zorder=0,
-                       add_labels=False,
-                       add_colorbar=False)
-
-# Add color bar
+# Add minor tick marks
+gv.add_major_minor_ticks(ax,
+                         x_minor_per_major=4,
+                         y_minor_per_major=4,
+                         labelsize=14)
+
+# Add and customize colorbar
 cbar_ticks = np.arange(24, 29.1, .3)
```

Comment:
```suggestion
cbar_ticks = np.arange(24, 29, .3)
```
I know you didn't edit this part, but for some reason the endpoint of this range was being included in the colorbar. Just switching it to 29 will resolve this.